go/signedexchange/README.md

```diff
@@ -148,11 +148,50 @@ You can dump the content of your sxg file by `dump-signedexchange`. If you want
 dump-signedexchange -i example.org.hello.sxg
 ```
 
-If `-verify` command-line flag is specified, `dump-signedexchange` checks if the signed exchange is valid.
+If the `-json` flag is passed, the output will be in JSON.
+
+```
+dump-signedexchange -i example.org.hello.sxg -json
+```
+
+You can also dump the content of a signed exchange from a URI. If you want to see the content of the signed exchange you're hosting at https://example.org/hello.html, run this command. By default this will request the latest version.
+
+```
+dump-signedexchange -uri https://example.org/hello.html
+```
+
+If the specified URI requires a special header to serve a signed exchange, you can pass request headers via the -requestHeader flag. The header key and value should be separated by a `:` and the value should be enclosed with double quotations `""`
```

Comment:
> the value should be enclosed with double quotations `""`

This might be confusing, as `dump-signedexchange` itself doesn't interpret quotations. Quotations may be needed by command-line shells, but exact rules for quotation / escaping could be different by shells (so it's probably out of scope of this document).

Reply:
I'm not sure if I'm reading the docs right, but the examples don't seem to match the docs—?

The examples seem to match curl's syntax (aside from the actual switch name), which seems like a good approach--well-understood by developers, and various tools seem to now output curl syntax for debugging (e.g. Chrome and Safari developer tools).

Reply:
@ithinkihaveacat The examples should match? Could you point out what specifically is confusing? After this PR the tool will be able to take sxg input from a curl command and also by passing the URI via flag